### PR TITLE
[2044] add created at and changed at timestamp for providers to api v1

### PR DESCRIPTION
**Pull request by defong**
### Context
v1 api providers endpoint
created at and changed at timestamp for providers

### Changes proposed in this pull request
Added created at and changed at timestamp for providers to api v1
Amended docs to reflect changes

### Guidance to review
`/api/v1/2019/providers`

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Add "created_at" & "changed_at" timestamps for providers to API V1](https://trello.com/c/zXJ8mDne/2044-add-created-at-timestamp-for-providers-to-api-v1)

File: app/serializers/provider_serializer.rb [hunk 1]
```diff
@@ -88,6 +88,14 @@ def recruitment_cycle
     object.recruitment_cycle.year
   end
 
+  def created_at
+    object.changed_at.iso8601
```
Comment (misaka) [inline app/serializers/provider_serializer.rb]: Shouldn't this be `created_at`?

Comment (misaka) [reply app/serializers/provider_serializer.rb]: Seems like the tests should ensure that these are two different timestamps to test thoroughly.

Also, did you check to see if it just Does The Right Thing if you don't even bother with these methods?

Comment (defong) [reply app/serializers/provider_serializer.rb]: "Fri, 30 Aug 2019 16:19:35 UTC +00:00"
vs
"2019-08-30T16:26:33Z" (iso)

**Pull request merged**